cygnus-arcgis/src/main/java/com/telefonica/iot/cygnus/backends/arcgis/restutils/NGSIArcgisFeatureTable.java

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright 2015-2017 Telefonica Investigación y Desarrollo, S.A.U
+ *
+ * This file is part of fiware-cygnus (FIWARE project).
+ *
+ * fiware-cygnus is free software: you can redistribute it and/or modify it under the terms of the GNU Affero
+ * General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.
+ * fiware-cygnus is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the
+ * implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License along with fiware-cygnus. If not, see
+ * http://www.gnu.org/licenses/.
+ *
+ * For those usages not covered by the GNU Affero General Public License please contact with iot_support at tid dot es
+ */
+package com.telefonica.iot.cygnus.backends.arcgis.restutils;
+
+import java.util.Date;
+
+import com.telefonica.iot.cygnus.backends.arcgis.model.Feature;
+import com.telefonica.iot.cygnus.log.CygnusLogger;
+
+/**
+ * 
+ * @author PMO Santander Smart City – Ayuntamiento de Santander
+ *
+ */
+public class NGSIArcgisFeatureTable extends ArcgisFeatureTable {
+
+    private static final String CLASS_NAME = "NGSIArcgisFeatureTable";
+    private CygnusLogger cygnusLogger;
```

Comment:
Is there any particular reason for you not to use the singleton logger on the project?

Reply:
> This model was designed to be a PDI (kettle) plugin, and BasseLogger was the way to make logs work both Cygnus and PDI. Finally we make an specific project for cygnus, and changed logs to work Cygnus-like.